.github/workflows/release.yml

```diff
@@ -20,14 +20,27 @@ on:
 
 jobs:
   prepare:
-    if: github.repository_owner == 'viamrobotics' && github.ref == 'refs/heads/main' && contains(fromJson('["njooma"]'), github.actor)
+    if: github.repository_owner == 'viamrobotics' && github.ref == 'refs/heads/main')
     runs-on: [self-hosted, x64]
     container:
       image: ghcr.io/viamrobotics/canon:amd64
     outputs:
       sha: ${{ steps.commit.outputs.commit_long_sha }}
       version: ${{ steps.bump_version.outputs.version }}
     steps:
+      - name: Check if organization member
```

Comment:
Is there no way to check if they're a team member, rather than an org member?

Reply:
updated to check team member

Reply:
update to update - reverted to this because github workflows evaluate weird